icons/square/48/terminus.svg

```diff
@@ -0,0 +1,20 @@
+<svg width="48" height="48" version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
That's how it is with all Circle icons. Probably because the Circle icons were treated with svgcleaner (and respective options) and new Square icons (well, new Circle ones as well) are optimised from Inkscape (using Scour). My Inkscape Scour options probably cause height/width variables not to come back.

Anyway, all icons show up properly with Qt, rsvg and Web SVG rendering engines.